packages/turbo-repository/scripts/publish.mjs

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
+
+import path from "path";
+import { execa } from "execa";
+import { copy } from "fs-extra";
+import { readFile, writeFile } from "fs/promises";
+
+const cwd = process.cwd();
+let platform = process.argv[process.argv.length - 1];
+
+if (platform.endsWith("publish.mjs")) {
+  // No platform was passed, default to dev platform
+  platform = "darwin-arm64";
```

Comment:
```suggestion
  platform = `${process.platform}-${process.arch}`
```

not sure if this works across the board

Reply:
It will not work across the board, linux in particular requires a libc specifier. However, for now at least, this is only used for local dev.